From 3197971754881fd64c207b418b4f0452053907f9 Mon Sep 17 00:00:00 2001
Message-Id: <3197971754881fd64c207b418b4f0452053907f9.1658235456.git.maciek.borzecki@gmail.com>
In-Reply-To: <cover.1658235456.git.maciek.borzecki@gmail.com>
References: <cover.1658235456.git.maciek.borzecki@gmail.com>
From: Paolo Pisati <paolo.pisati@canonical.com>
Date: Tue, 27 Oct 2020 09:45:23 +0000
Subject: [PATCH 5/5] UBUNTU: SAUCE: apparmor: rename kzfree() to
 kfree_sensitive()

Upstream-Status: Inappropriate [not author]
The patch was imported from Ubuntu Kernel git server
(https://git.launchpad.net/~ubuntu-kernel/ubuntu/+source/linux/+git/jammy) as
of commit id e301d0f4cc619e1090e97b5eda4e005ac08c2576.

Signed-off-by: Paolo Pisati <paolo.pisati@canonical.com>
Signed-off-by: Maciek Borzecki <maciek.borzecki@gmail.com>
---
 security/apparmor/policy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/apparmor/policy.c b/security/apparmor/policy.c
index a00e39b5c7b28a8eb75ffdbf7349c6deb5e67766..f03726d962cc34d8038c162c18cebf289c0cb1e3 100644
--- a/security/apparmor/policy.c
+++ b/security/apparmor/policy.c
@@ -222,7 +222,7 @@ void aa_free_profile(struct aa_profile *profile)
 	aa_free_file_rules(&profile->file);
 	aa_free_cap_rules(&profile->caps);
 	aa_free_rlimit_rules(&profile->rlimits);
-	kzfree(profile->net_compat);
+	kfree_sensitive(profile->net_compat);
 
 	for (i = 0; i < profile->xattr_count; i++)
 		kfree_sensitive(profile->xattrs[i]);
-- 
2.37.1

